lib/reel/session.rb

```diff
@@ -55,17 +56,21 @@ def finalize_session
         make_header @bag.save
       end
 
+      def session_config options={}
+        Reel::Session.configuration(self.connection.server,options)
```

Comment:
since session configuration is not supposed to change during a server lifetime (is it?), i think we could do something like this:

```ruby
@_session_config ||= Reel::Session.configuration self.connection.server, options
```

see thoughts below on `@config`...